deepchecks/tabular/checks/data_integrity/percent_of_nulls.py

```diff
@@ -0,0 +1,109 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module with 'PercentOfNulls' check."""
+import typing as t
+
+import plotly.express as px
+
+from deepchecks.core.check_result import CheckResult
+from deepchecks.core.checks import DatasetKind, ReduceMixin
+from deepchecks.core.condition import ConditionCategory, ConditionResult
+from deepchecks.tabular import Context, SingleDatasetCheck
+from deepchecks.utils.strings import format_list, format_percent
+from deepchecks.utils.typing import Hashable
+
+__all__ = ['PercentOfNulls']
+
+
+TPercentOfNulls = t.TypeVar('TPercentOfNulls', bound='PercentOfNulls')
+
+
+class PercentOfNulls(SingleDatasetCheck, ReduceMixin):
```

Comment:
Do we need a plot example for this? @shir22 @noamzbr

Reply:
added